_config.yml

```diff
@@ -59,5 +59,7 @@ navigation:
     link: /patents.html
   - title: Sponsors
     link: /sponsors.html
+  - title: Software
```

Comment:
I think this should go after news or before Sponsors. It looks odd being the last item in the navigation bar

Reply:
This is now after patents but before sponsors